env.wochenende.minimal.yml

```diff
@@ -1,4 +1,4 @@
-name: wochenende
+name: miniWE
```

Comment:
It is confusing to change the wochenende env name. We would need to change many scripts and docu to make this clear.